src/app/operations/operations-db.service.spec.ts

```diff
@@ -71,6 +71,50 @@ describe('OperationDbService', () => {
     });
   }));
 
+  it('should add a supplier', fakeAsync(() => {
```

Comment:
`fakeAsync` is used when we call `tick()`, you should use `async` when only calling `.then()`